compute/cker/include/cker/operation/optimized/BinaryArithmeticOps.h

```diff
@@ -450,6 +569,27 @@ inline void BroadcastMulFivefold(const BinaryArithmeticOpParam &params,
   }
 }
 
+inline void BroadcastMulDispatchQuant8(const BinaryArithmeticOpParam &params,
+                                       const Shape &input1_shape, const uint8_t *input1_data,
+                                       const Shape &input2_shape, const uint8_t *input2_data,
+                                       const Shape &output_shape, uint8_t *output_data)
+{
+  if (params.broadcast_category == BroadcastableOpCategory::kGenericBroadcast)
+  {
+    const std::function<uint8_t(const BinaryArithmeticOpParam &, const uint8_t &, const uint8_t &)>
+        fn = [](const BinaryArithmeticOpParam &params, const uint8_t &a,
+                const uint8_t &b) -> uint8_t {
+      return static_cast<uint8_t>(quant8_mul(params, a, b));
+    };
+    reference::BroadcastBinaryArithmeticOpSlowQuant8(params, input1_shape, input1_data,
```

Comment:
This function call has a dependency to pull request #2345 , hence the build errors.